### PR TITLE
feat(dashboard): Empty workflow list state & improvements

### PR DESCRIPTION
**Pull request by desiprisg**
### What changed? Why was the change needed?

* Empty workflow list state
* min height on dashboard layout children
* Hide table footer if not necessary
* Overflow now happens outside table as discussed privately with @SokratisVidros 
* Updated tanstack query to v5 as it was on v4
### Screenshots

image

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

File: apps/dashboard/src/components/workflow-cloud.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+      </g>
+      <circle cx="21.5" cy="19" r="5" fill="#FBFBFB" />
+      <circle cx="18.5" cy="109" r="7" fill="#FBFBFB" />
+      <circle cx="145.5" cy="35" r="7" fill="#FBFBFB" />
+      <circle cx="134.5" cy="8" r="4" fill="#FBFBFB" />
+      <g filter="url(#filter1_b_2411_21338)">
+        <path
+          d="M76 85.5C76 72.2452 86.7452 61.5 100 61.5C113.255 61.5 124 72.2452 124 85.5C124 98.7548 113.255 109.5 100 109.5C86.7452 109.5 76 98.7548 76 85.5Z"
+          fill="#99A0AE"
+          fill-opacity="0.1"
```
Comment (LetItRock) [inline apps/dashboard/src/components/workflow-cloud.tsx]: please update all attributes with dashes

File: apps/dashboard/src/components/workflow-cloud.tsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
+import { HTMLAttributes } from 'react';
+
+type WorkflowCloudProps = HTMLAttributes<HTMLOrSVGElement>;
+export const WorkflowCloud = (props: WorkflowCloudProps) => {
+  return (
+    <svg width="153" height="120" viewBox="0 0 153 120" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
```
Comment (LetItRock) [inline apps/dashboard/src/components/workflow-cloud.tsx]: did you run this through minification? https://jakearchibald.github.io/svgomg/

File: apps/dashboard/src/components/workflow-list.tsx [hunk 3]
```diff
@@ -47,16 +51,45 @@ export const WorkflowList = () => {
...
+            No workflows exist, create workflows to orchestrate notifications
+          </span>
+          <p className="text-foreground-600 max-w-[55ch] text-sm">
+            Workflows in Novu handle event-driven notifications across multiple channels in a single, version-controlled
+            flow, with the ability to manage preference for each subscriber.
+          </p>
+        </div>
+
+        <div className="flex items-center justify-center gap-6">
+          <Link to={'#'} className={buttonVariants({ variant: 'link', className: 'text-foreground-600 gap-1' })}>
```
Comment (LetItRock) [inline apps/dashboard/src/components/workflow-list.tsx]: link: https://docs.novu.co/concepts/workflows

**Review by LetItRock — APPROVED** [review]
🙌

File: apps/dashboard/src/components/workflow-cloud.tsx (hunk 2 above)
Comment (desiprisg) [reply apps/dashboard/src/components/workflow-cloud.tsx]: I did not

Comment (rifont) [reply apps/dashboard/src/components/workflow-cloud.tsx]: Neat tool!

**Pull request merged**